virus/H_metis.md

```diff
@@ -0,0 +1,132 @@
+
+## Results
+
+**Initial detection of Hemiptinidovirus metis**
+The index case reporting the presence of this virus comes from a research group at the University of Connecticut in 2017. This group generated RNA sequencing data of the keeled treehopper (Entylia carinata) and its close leafhopper relative, the glassy-winged sharpshooter (Homalodisca vitripennis), using Illumina sequencing. The virus was found in 29 sequencing runs across both species. They later published a study containing this transcriptomics data in 2020, titled “Co-option of wing-patterning genes underlies the evolution of the treehopper helmet” [@Fisher2020]. This dataset was generated in order to explore the developmental origin of the treehopper insect's "helmet". The transcriptomes of treehoppers and leafhoppers were compared to understand their divergence overtime and the development of the "helmet" [@Fisher2020].
+
+**Subsequent reports of Hemiptinidovirus metis**
+The virus has since been identified in other sequencing runs. One project by the same group from the University of Connecticut detected the virus in 43 Illumina sequencing runs, spanning the same two species of treehopper and leafhopper. Another project out of Peking University identified the virus in 10 Illumina sequencing runs in another leafhopper species Nephotettix cincticeps, also found a sequencing run from Japan’s Agrobiological Sciences institute. Other studies have identified the virus in the red-streaked leafhopper (Balclutha rubrostriata), as well as Xibalbanus tulumensis, a venomous crustacean, and Citrus maxima, a citrus fruit. All run information above was identified from the SRA, where all sequences were pulled from in a global screen [@Sayers2022].
+
+**Ecology of host species**
+The majority of sequencing results were found in species of treehoppers and leafhoppers. These species of leafhoppers and treehoppers are native to different parts of the world such as North America, South America, and Asia, but in general these insects have been introduced to other parts of the world and are found globally [@Matsumoto2018]. Leafhoppers are known to be viral vectors, carrying viruses that can be passed onto plants, such as the Rice rice tungrovirus (from the order Picornavirales), Rice dwarf virus (from the order Reovirales), and Potato yellow dwarf virus (from the order Mononegavirales) [@Bhat2020]. Some leafhoppers including Homalodisca vitripennis can be found on citrus trees, potentially explaining how a citrus fruit species contained the virus [@Williams2017]. It remains unclear how a crustacean only found in the caves of the Yucatan peninsula in Mexico could have obtained the virus [@VR2014], but it is likely that species of leafhoppers are found around this area.
+
+### Virus Genome {Q3}
+
+![image](https://github.com/vedikajha/VIRUSxDISCVRY/assets/116615772/3736765b-76eb-4308-926c-738e9b5d4aba)
```

Comment:
The images should be part of the PR, not a link to another repo.

To this Pull Request, all the images files should be added in the directory `virus/img/h_metis/<image.png>` and the links corrected as such.